packages/components/inputs/radio-input/src/radio-option.tsx

```diff
@@ -111,22 +127,20 @@ const Option = (props: TOptionProps) => {
           aria-readonly={props.isReadOnly}
           {...filterDataAttributes(props)}
         />
-        <div css={getContainerStyles(props)}>
-          {props.isChecked ? (
-            <RadioOptionCheckedIcon size="medium" />
-          ) : (
-            <RadioOptionUncheckedIcon size="medium" />
-          )}
+        <div css={getContainerStyles(stylesProps)}>
+          <RadioOptionBorder {...stylesProps}>
+            {props.isChecked ? <RadioOptionKnob {...stylesProps} /> : null}
+          </RadioOptionBorder>
```

Comment:
This is the major change that is applied to both the old and the new theme. We ditch svgs in favour of a bunch of styled divs.

My additional question is - should I delete the `icons` folder? Or leave it for backwards compatibility